### PR TITLE
 DEV-2473: Add Chromatic

### PR DESCRIPTION
**Pull request by nrh-cklimas**
Please fill out each section even if it's just with "N/A"

#### Plan? (if this draft/incomplete indicate what you intend to do and how)

n/a

#### What's this PR do?

This connects our repo to Chromatic. I also added a line to use NPM caching in our JS test workflow, to speed it up.

#### Why are we doing this? How does it help us?

We hope it'll speed up design reviews and generally facilitate collaboration.

#### How should this be manually tested? Please include detailed step-by-step instructions.

n/a, this is not user-facing

#### Did this PR make changes to the user interface that may require changes to the user-facing docs?

No.

#### Have automated unit tests been added? If not, why?

No, this adds a GitHub action.

#### How should this change be communicated to end users?

n/a

#### Are there any smells or added technical debt to note?

n/a

#### Has this been documented? If so, where?

No.

#### What are the relevant tickets? Add a link to any relevant ones.

https://news-revenue-hub.atlassian.net/browse/DEV-2473

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?

No.

#### Are there next steps? If so, what? Have tickets been opened for them? List next-step tickets here:

No.

### CODE REVIEW

**Review by GuiMend — APPROVED** [review]
Looks good 👍 
How can I test chromatic? Do we have a pattern URL to check or do I have to login to chromatic (in this case, do I need an invite for credentials)?

**Comment by nrh-cklimas** [comment]
@GuiMend I think the answer is you'd log into Chromatic, but I'm not certain. I'm going to invite the team to it once this PR merges.

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Copied from https://www.chromatic.com/docs/github-actions
+
+# List of jobs
+jobs:
+  chromatic-deployment:
+    # Operating System
+    runs-on: ubuntu-latest
+    # Job steps
+    steps:
+      - uses: actions/checkout@v1
```
Comment (x110dc) [inline .github/workflows/chromatic.yml:18]: I'm no Actions expert, and maybe there's a reason to pin it to v1, but all of our other workflows use at least v3 of the `checkout` action. I guess I'd probably change it to v3 and see if it works. If not, I'd set it back to v1.

**Review by x110dc — APPROVED** [review]
Asked one minor question. Feel free to ignore it and proceed if it doesn't make sense.

File: .github/workflows/chromatic.yml (hunk 1 above)
Comment (nrh-cklimas) [reply .github/workflows/chromatic.yml]: Good catch--Chromatic gives you [a template GH action](https://www.chromatic.com/docs/github-actions) but it's out-of-date. Updated in 59df2a7ba959d19b052e1680aabafacab5ff6f5f

**Review by x110dc — APPROVED** [review]

**Pull request merged**